<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bouton_reclamation</class>
 <widget class="QDialog" name="bouton_reclamation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_16">
    <attribute name="title">
     <string>Réclamation</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>581</width>
       <height>341</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;kjknjj&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
 #MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid red;
border-radius: 8px;
}

QPushButton {
color: white;
  background: bleu;

border-width: 6px;
border-color: grey;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 150px;
max-width: 150px;
min-height: 35px;
max-height: 35px;
}
QLabel {
     border: 4px solid black;
     border-radius: 12px;
     padding: 0 12px;
     background: red;
     selection-background-color: black;
 }
QPushButton:hover { background-color: #FF0101; }
 
  

  

 
 </string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_17">
      <attribute name="title">
       <string>Créer</string>
      </attribute>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>titre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>sujet</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_titrerec_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_daterec_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sujetrec_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inserrec_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>Insérer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_18">
      <attribute name="title">
       <string>Consulter</string>
      </attribute>
      <widget class="QTableView" name="tableView_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_19">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QTableView" name="tableView_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifierev_5">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>210</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>160</width>
         <height>194</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="verticalSpacing">
         <number>40</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>titre</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>sujet</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>date</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_titrerec_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_daterec_4"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_sujetrec_4"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_20">
      <attribute name="title">
       <string>Eliminer</string>
      </attribute>
      <widget class="QLabel" name="label_eliminerrec_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_eliminerrec_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimerrec_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>100</y>
         <width>172</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2">
    <attribute name="title">
     <string>Evenement</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
